### PR TITLE
SS upload: add documentation and example scripts

### PR DESCRIPTION
**Pull request by jraddaoui**
Connects to https://github.com/archivematica/Issues/issues/688.

Related to work done in https://github.com/artefactual/automation-tools/pull/109.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -477,66 +477,87 @@ cd /usr/lib/archivematica/automation-tools/
...
+argument, another METS file will be generated alongside the objects folder
+containing only a reference to the ZIP file (without AMD or DMD sections), used
+to upload the DIP to an AtoM instance; or the same AIP METS file will be placed
+alongside the objects folder, used for uploads to the Storage Service.
 
 While `aips/create_dip.py` only processes one AIP per execution,
 `aips/create_dips_job.py` will process all AIPs in a given Storage Service
-location, keeping track of them in an SQLite database. Both scripts require 7z
-installed and available to extract the AIPs downloaded from the Storage Service.
+location, keeping track of them in an SQLite database. The later also accepts
```
Comment (sallain) [inline README.md]:
```suggestion
location, keeping track of them in an SQLite database. The latter also accepts
```

File: README.md [hunk 2]
```diff
@@ -477,66 +477,87 @@ cd /usr/lib/archivematica/automation-tools/
...
+different subsets of parameters to automatically upload the created DIPs to the
+Storage Service or to an AtoM instance. Both scripts require 7z installed and
```
Comment (sallain) [inline README.md]:
```suggestion
Storage Service or to an AtoM instance. Both scripts require 7z to be installed and
```

File: README.md [hunk 3]
```diff
@@ -477,66 +477,87 @@ cd /usr/lib/archivematica/automation-tools/
...
+available to extract the AIPs downloaded from the Storage Service.
+
+As mentioned, this DIPs can be uploaded to AtoM or the Storage Service using
```
Comment (sallain) [inline README.md]:
```suggestion
As mentioned, these DIPs can be uploaded to AtoM or the Storage Service using
```

File: README.md [hunk 4]
```diff
@@ -477,66 +477,87 @@ cd /usr/lib/archivematica/automation-tools/
...
+`dips.storage_service_upload` and `dips.atom_upload` or as part of the
+`aips/create_dips_job.py` execution.
+
+The AtoM upload requires a passwordless SSH connection to the AtoM host for the
+user running the script and the AtoM host has to be already added to list of
```
Comment (sallain) [inline README.md]:
```suggestion
user running the script. The AtoM host must be added to list of
```

File: README.md [hunk 5]
```diff
@@ -477,66 +477,87 @@ cd /usr/lib/archivematica/automation-tools/
...
+known hosts. [More info][Send DIPS].
+
+The Storage Service upload requires access to the Archivematica pipeline's
+currently processing location path (the shared path), to copy the DIP folder in
```
Comment (sallain) [inline README.md]:
```suggestion
currently processing location path (the shared path) in order to copy the DIP folder 
```

File: README.md [hunk 6]
```diff
@@ -477,66 +477,87 @@ cd /usr/lib/archivematica/automation-tools/
...
+there before sending a requests to the Storage Service to process that DIP.
```
Comment (sallain) [inline README.md]:
```suggestion
before sending a requests to the Storage Service to process that DIP.
```

**Comment by jraddaoui** [comment]
Done, thanks a lot @sallain!

**Pull request merged**